tasks/eva_3190/src/main/groovy/eva3190/eva3190_deprecate_remapped_nra_sves.groovy

```diff
@@ -0,0 +1,127 @@
+package eva3190
+
+import eva3190.RemappingOutputParams
+import eva3190.RemediateDuplicateRemappedSS
+import eva3190.DeprecateRemappedNraSS
+import groovy.cli.picocli.CliBuilder
+import org.slf4j.LoggerFactory
+import org.springframework.batch.test.MetaDataInstanceFactory
+import uk.ac.ebi.eva.accession.deprecate.Application
+import uk.ac.ebi.eva.commons.batch.io.AggregatedVcfLineMapper
+import uk.ac.ebi.eva.commons.batch.io.VcfReader
+import uk.ac.ebi.eva.commons.core.models.Aggregation
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant
+import uk.ac.ebi.eva.remapping.ingest.batch.processors.VariantToSubmittedVariantEntityRemappedProcessor
+
+import static org.springframework.data.mongodb.core.query.Criteria.where
+import static org.springframework.data.mongodb.core.query.Query.query
+import static uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment.*
+
+// This script deprecates variants remapped from variants due to NRA (novel-reference allele) variants
+// generated during remapping - see also https://www.ebi.ac.uk/panda/jira/browse/EVA-2919
+def cli = new CliBuilder()
+cli.propertiesFile(args: 1, "Properties file to use for deprecation", required: true)
+cli.remappedAssembly(args: 1, "Remapped assembly", required: true)
+cli.remappedVcfDir(args: 1, "Directory with the remapped variants with NRA alleles", required: true)
+def options = cli.parse(args)
+if (!options) {
+    cli.usage()
+    System.exit(1)
+}
+
+// Create a connection to the data source with a maximum pool size 1 i.e., one connection
+def dbEnv = createFromSpringContext(options.propertiesFile, Application.class,
+        ["parameters.assemblyAccession": options.remappedAssembly, "spring.datasource.hikari.maximumPoolSize": 1])
+def logger = LoggerFactory.getLogger(Application.class)
+// Construct variant to SVE processor employed by the remapping ingestion job
+def variantToSve = new VariantToSubmittedVariantEntityRemappedProcessor(options.remappedAssembly, "dummySrc", "dummyHash")
```

Comment:
dummySrc, dummyHash good enough for remapping attributes :) because we only need the actual Submitted variant contents from this remapping processor...